apps/documentation/public/polyfill-db.json

```
@@ -0,0 +1,11 @@
+{
+		"polyfills": [
+				{ 
+						"identifier": "HTMLElement",
+						"imports": ["@lit-labs/ssr-dom-shim", "HTMLElement"]
```

The names `imports` suggest a list however this array is referencing to a package name and a named export so IMHO is confusing.

```suggestion
             "import":  {  
						   "package": "@lit-labs/ssr-dom-shim", 
						   "name": "HTMLElement"   
						 }
```

---

Yeah. Better to use `import`